internal/registry/json.go

```diff
@@ -135,8 +134,8 @@ func newImage(baseURL string, manifest *imageManifestV1, config *imageConfigV1)
 		}
 		h := history[k]
 		k++
-		if !isMediaTypeImageLayerV1(l.MediaType) {
-			continue // skip unsupported media types
+		if l.MediaType == "application/vnd.docker.image.rootfs.foreign.diff.tar.gzip" {
```

Comment:
is there a reason for not defining const for this foreign layer?

Reply:
yeah because it is only used once... should I do that?

Reply:
well, up to you! either is fine to me, let's make it const when another usage comes up!